pkg/collector/ibc_collector.go

```diff
@@ -24,8 +24,10 @@ var (
 		"Returns light client expiry in unixtime.",
 		[]string{
 			"host_chain_id",
-			"client_id",
 			"target_chain_id",
+			"src_chain_name",
```

Comment:
There is a bit inconsistency with `host_chain_id` and `src_chain_name`, maybe better to change also `host_chain_id` -> `src_chain_id` and `target_chain_id` -> `dst_chain_id`?
Also docs with metrics samples needs to be updated if we adding new labels.